Helper/Entity/ProductHelper.php

```diff
@@ -728,7 +743,7 @@ private function addImageData(array $customData, Product $product, $additionalAt
         return $customData;
     }
 
-    private function addInStock($defaultData, $customData, Product $product)
+    protected function addInStock($defaultData, $customData, Product $product)
```

Comment:
There is a reason why we are altering the visibility of this method?

Reply:
To be overriden by the inventory module, as discussed ;)